styles/style.css

```diff
@@ -120,8 +161,16 @@ p {
 /* hero */
 
 .hero {
-  background-image: url("header-bike.jpg");
+  background-image: url("/bikes-for-refugees/images/header-bike.jpg");
   background-color: var(--grey-light);
+  height: 58%;
+  background-size: auto;
+  color: white;
+  padding-left: 4.4%;
```

Comment:
This seems a bit like a kind of "eyeball" number or "magic number". What made you decide on 58% and 4.4%?